miso-web/src/main/webapp/scripts/box_visualization.js

```diff
@@ -20,686 +20,696 @@
+      }
+      var row = event.data.row;
+      var items = [];
+      for (var col = 1; col <= self.size.cols; col++) {
+        items.push(self.position[row][col]);
+      }
+      toggleGroupSelection(event, items);
+    };
 
-  self.selectCol = function(event) {
-    var col = event.data.col;
-    var items = [];
-    for (var row = 1; row <= self.size.rows; row++) {
-      items.push(self.position[row][col]);
+    self.selectCol = function(event) {
+      if (!self.isMultiSelectEnabled()) {
+        return;
+      }
+      var col = event.data.col;
+      var items = [];
+      for (var row = 1; row <= self.size.rows; row++) {
+        items.push(self.position[row][col]);
+      }
+      toggleGroupSelection(event, items);
+    };
+
+    self.selectPos = function(row, col) {
+      self.clearSelection();
+      self.select(self.position[row][col], false);
     }
-    toggleGroupSelection(event, items);
-  };
 
-  self.selectPos = function(row, col) {
-    self.clearSelection();
-    self.select(self.position[row][col], false);
-  }
-
-  self.getBoxPosition = function(row, col, tCell) {
-    var opts = self.getBoxPositionOpts(row, col);
-    opts.row = row;
-    opts.col = col;
-    opts.cell = tCell;
-    opts.parentVisual = self;
-    return new BoxPosition(opts);
-  };
+    self.getBoxPosition = function(row, col, tCell) {
+      var opts = self.getBoxPositionOpts(row, col);
+      opts.row = row;
+      opts.col = col;
+      opts.cell = tCell;
+      opts.parentVisual = self;
+      return new BoxPosition(opts);
+    };
 
-  self.onSelectionChanged = function(items) {
-  };
+    self.onSelectionChanged = function(items) {
+    };
 
-  self.setDisabled = function(disable) {
-    disabled = disable;
-  };
+    self.setDisabled = function(disable) {
+      disabled = disable;
+    };
 
-  return self;
-};
-
-var Box = Box || {};
-
-Box.DialogVisual = function(scan) {
-  var self = new BoxVisual();
-  self.scan = scan;
-
-  self.show = function(opts) {
-    jQuery('#dialogInfoAbove').html(opts.infoabove);
-    jQuery('#dialogInfoBelow').html(opts.infobelow);
-    jQuery('#dialogDialog').dialog({
-      autoOpen: false,
-      title: 'Scan',
-      width: Box.dialogWidth,
-      modal: true,
-      resizable: false,
-      position: [jQuery(window).width() / 2 - Box.dialogWidth / 2, 50],
-      buttons: {}
-    });
-    self.create({
-      div: '#dialogVisual',
-      size: {
-        rows: self.size.rows,
-        cols: self.size.cols
-      },
-      data: self.scan.items
-    });
-    jQuery('#updateSelected, #removeSelected, #emptySelected').prop('disabled', true).addClass('disabled');
-    jQuery('#dialogDialog').dialog('open');
-  };
+    self.getRowLabel = function(row) {
+      return Box.utils.getRowLabel(row);
+    };
 
-  self.getBoxPositionOpts = function(row, col) {
-    var pos = Box.utils.getPositionString(row, col);
-    var boxables;
+    self.getColLabel = function(col) {
+      return Box.utils.getColLabel(col);
+    },
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove use of this comma operator. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS878)